### DISCUSSION TITLE
Send midi messages to an external device

### DISCUSSION THREAD

**Discussion by denybear** [FluidSynth/fluidsynth#1198] [Q&A] [answered]
Hello,
Fluidsynth can receive some MIDI messages from external devices; I have successfully implemented a fluid midi driver to get midi messages from my external midi controller device.
Is there a way for Fluidsynth to **send** MIDI messages to an external device ? I would like to switch on/off leds of my controller device, this can be done only by sending the proper midi message to the hardware.
Thanks for your support !

**Comment by derselbst** [comment, marked as answer]
> Is there a way for Fluidsynth to send MIDI messages to an external device ?

No that's not possible, sorry. Consider using a MIDI routing software to redirect / manipulate the messages from your external midi controller for controlling the LEDs.
